tests/ci/common_posix_setup.sh

```diff
@@ -140,8 +156,31 @@ function fips_build_and_test {
 }
 
 function build_and_test_valgrind {
-  run_build "$@"
-  run_cmake_custom_target 'run_tests_valgrind' && run_cmake_custom_target 'run_ssl_runner_tests_valgrind'
+  if [[ $PLATFORM == "aarch64" ]]; then
+    run_build "$@" -DCMAKE_C_FLAGS="$VALGRIND_STATIC_CAP_FLAGS"
+    run_cmake_custom_target 'run_tests_valgrind'
+
+    # Disable all capabilities and run again
+    # (We don't use the env. variable OPENSSL_armcap because it is currently
+    #  restricted to the case of runtime discovery of capabilities
+    #  in cpu_aarch64_linux.c)
+    run_build "$@" -DCMAKE_C_FLAGS="-DOPENSSL_STATIC_ARMCAP"
+    run_cmake_custom_target 'run_tests_valgrind'
+  else
+    run_build "$@"
+    run_cmake_custom_target 'run_tests_valgrind'
+  fi
+}
+
+function build_and_test_ssl_valgrind {
```

Comment:
This only seems to be running the ssl runner tests in valgrind , while `build_and_test_valgrind` still calls both `crypto_test` and `ssl_test`.
With that in context, could we rename this into something clearer? I got a bit confused and thought this was running `ssl_test` as well. `build_and_test_ssl_runner_valgrind` might be good.

Reply:
Thanks, Samuel, for the clarification. Done.